packages/common/src/elevate.ts

```diff
@@ -33,21 +45,33 @@ function createShadow({ level, color, depth, spread }) {
   return shadows.slice(0, depth).join();
 }
 
-function elevate({
+interface ElevateProps {
+  color?: string;
+  level?: number;
```

Comment:
**suggestion**: since this only supports up to 3, shouldn't our types reflect this?

quick and dirty way would be to do this (we could build a Range util later):
```suggestion
  level?: 0 | 1 | 2 | 3 
```